src/components/Form/__tests__/CheckboxWrapper.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import { shallow } from 'enzyme'
+
+import { Form } from '../'
+
+describe('<Form.CheckboxWrapper id="checkbox-1" />', () => {
+  it('renders the appropriate classNames', () => {
+    const $ = shallow(<Form.CheckboxWrapper className="test" />)
+    expect($.hasClass('c-form-input test')).toBe(true)
+  })
+
+  it('renders children', () => {
```

Comment:
Let's test that children get rendered in the correct order like in `RadioWrapper` as the ordering is critical for functionality.